packages/mui-codemod/README.md

```diff
@@ -1075,43 +1101,29 @@ npx @mui/codemod@next deprecations/pagination-item-props <path>
 npx @mui/codemod@next deprecations/slider-props <path>
 ```
 
-#### `toggle-button-group-classes`
+#### `step-connector-classes`
```

Comment:
This and the changes below are just caused by sorting alphabetically the sections.